src/mqttcm_connect.c

```diff
@@ -414,6 +416,49 @@ int validateForMqttInit()
 	return 0;
 }
 
+void custom_log_callback(struct mosquitto *mosq, void *userdata, int level, const char *message)
+{
+    // Control logging based on levels
+    if (level == MOSQ_LOG_DEBUG) {
+        // Log debug messages
```

Comment:
Remove this additional comments and keep only "// Control logging based on levels"

Reply:
Removed